src/utils.ts

```diff
@@ -17,3 +17,11 @@ export function extractAxiosErrorMessage(err: AxiosError): string {
 
   return errMsg;
 }
+
+export function isHexString(string: string) {
+  if (typeof string !== 'string') {
```

Comment:
As we are using TypeScript, we can obviate types instead of checking them dynamically

Reply:
fixed